apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -1,4175 +1,1000 @@
-guy
-guys
-gyro
+gp
+gq
+gr
+gs
+gt
+gu
+gw
+gy
 h
-habit
-habits
 had
 hadn't
-hair
+hadnt
 half
-halign
-hall
-ham
-hand
-handed
-handle
-handled
-handling
-hands
-hang
-hanging
-happen
-happened
-happening
 happens
-happily
-happiness
-happy
-hard
-harder
 hardly
-harm
-harmony
 has
 hasn
 hasn't
-hat
-hate
-hated
-hatred
+hasnt
 have
+haven
 haven't
+havent
 having
 he
 he'd
 he'll
 he's
-head
-headed
-heading
-headquarters
-heads
-health
-healthy
-hear
-heard
-hearing
-heart
-hearts
-heat
-heaven
-heavily
-heavy
-heels
-height
-heights
-held
-hell
+hed
 hello
-help
-helped
-helpful
-helping
-helpless
-helps
-hen
 hence
 her
-herd
 here
 here's
 hereafter
 hereby
 herein
+heres
 hereupon
-heritage
-hero
-heroic
 hers
+herse
+herse"
 herself
-hesitated
+herse”
+hes
 hi
-hidden
-hide
-high
-higher
-highest
-highly
-hill
 him
+himse
+himse"
 himself
-hired
+himse”
 his
-historian
-historians
-historic
-historical
-history
-hit
 hither
-hits
-hold
-holder
-holding
-holds
-hole
-holes
-hollywood
+hk
+hm
+hn
 home
-homes
-hon
-honest
-honey
-honor
-honored
-hope
-hoped
+homepage
 hopefully
-hopes
-hoping
-horizon
-horse
-horses
-hosp
-host
-hot
-hotels
-hour
-hours
-house
-household
-houses
-housing
 how
+how'd
+how'll
 how's
 howbeit
 however
 hr
-href
+ht
+htm
 html
 http
-huge
-human
-humanity
+hu
 hundred
-hundreds
-hung
-hungry
-hunt
-hunting
-hurried
-hurry
-husband
-hydrogen
-hypothalamic
 i
-I
 i'd
 i'll
 i'm
 i've
-ice
-idea
-ideal
-ideas
-identical
-identification
-identified
-identify
-identity
-ideological
+i.e
+i.e.
+id
 ie
 if
-ignored
-illness
-illusion
-illustration
-image
-images
-imagination
-imagine
-imagined
-imitation
-immediate
-immediately
-impact
-implications
-importance
-important
-imposed
-impossible
-impressed
-impression
-impressions
-impressive
-improve
-improved
-improvement
-improvements
-impulse
+ii
+il
+ill
+im
 in
-inadequate
 inasmuch
 inc
 inc.
-inch
-inches
-incident
-inclined
-include
-included
-includes
-including
-income
-increase
-increased
-increases
-increasing
-increasingly
-incredible
 indeed
-index
 indicate
 indicated
 indicates
-indication
-indirect
-individual
-individuals
-industrial
-industry
-inevitable
-inevitably
-influence
-information
-informed
-inherent
-initial
-initiative
-injured
-injury
-inner
-innocence
-innocent
-input
-insect
-insects
-inside
-insight
-insist
-insisted
 insofar
-insp
-inspection
-inspired
-installed
-instance
-instances
-instant
 instead
-institution
-institutions
-instruction
-instructions
-instrument
-instruments
-insurance
-insure
-integration
-intended
-intense
-intensity
-intention
-intentions
-interest
-interested
-interesting
-interests
-interference
-interior
-internal
-international
-interpretation
-interpreted
-intervals
-intervention
-interview
-interviews
-intimate
+int
 into
-introduced
-introduction
-invariably
-invent
-invention
-inventory
-investigation
-investigations
-investment
-invited
-involve
-involved
-involves
-involving
-inward
-iron
+io
+iq
```

Comment:
It looks like the new list has a full list of country-level TLDs in it. I'd skip adding those as, for example, `.iq` for Iraq could accidentally filter out IQ the intelligence quotient, at least for the first pass. We might find out that we need to re-add them later though.